### PR TITLE
Update to version 0.9.0

### PR DESCRIPTION
**Pull request by m-rossi**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Not sure why it does not trigger the PR automatically.

Some notes:
* I did not include https://github.com/genotrance/px/blob/v0.9.0/setup.py#L82 as we only build for Python > 3.7

### CODE REVIEW

**Comment by m-rossi** [comment]
@conda-forge-admin, please rerender

**Comment by m-rossi** [comment]
@conda-forge-admin, please rerender

**Comment by m-rossi** [comment]
Looks good. I also enabled builds on Linux with success.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,7 +27,9 @@ requirements:
     - libcurl
     - netaddr
     - psutil
+    - pyspnego
     - python
```
Comment (fcollonval) [inline recipe/meta.yaml:31]: let's make the python constraint explicit

```suggestion
    - python >=3.8.0
```

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks a lot @m-rossi 

I have a minor suggestion. And we should be good to go.

**Comment by fcollonval** [comment]
Ok let's revert my suggestion as the linter does not like it.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,7 +27,9 @@ requirements:
     - libcurl
     - netaddr
     - psutil
-    - python
+    - pyspnego
+    - python >=3.8.0
```
Comment (fcollonval) [inline recipe/meta.yaml]:
```suggestion
    - python
```

**Pull request merged**